user-service/src/test/resources/schema.sql

```diff
@@ -0,0 +1,118 @@
+-- MySQL Script generated by MySQL Workbench
```

Comment:
Clean up the SQL schema so that it follows H2's syntax and then remove any unused tables for this service's tests e.g. flights
And remove the MYSQL comments as well.